manifests/0000_90_kube-apiserver-operator_06_api_clients_dashboard.yaml

```diff
@@ -0,0 +1,994 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: grafana-dashboard-api-client-go
+  namespace: openshift-config-managed
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: 'true'
+  labels:
+    console.openshift.io/dashboard: 'true'
+data:
+  api-client-go.json: |-
+    {
+      "annotations": {
+        "list": [
+          {
+            "builtIn": 1,
+            "datasource": "-- Grafana --",
+            "enable": true,
+            "hide": true,
+            "iconColor": "rgba(0, 211, 255, 1)",
+            "name": "Annotations & Alerts",
+            "target": {
+              "limit": 100,
+              "matchAny": false,
+              "tags": [],
+              "type": "dashboard"
+            },
+            "type": "dashboard"
+          }
+        ]
+      },
+      "editable": true,
+      "fiscalYearStartMonth": 0,
+      "gnetId": null,
+      "graphTooltip": 0,
+      "id": 4,
+      "iteration": 1639500561327,
+      "links": [],
+      "liveNow": false,
+      "panels": [
+        {
+          "aliasColors": {},
+          "bars": false,
+          "dashLength": 10,
+          "dashes": false,
+          "datasource": "$datasource",
+          "description": "",
+          "fieldConfig": {
+            "defaults": {
+              "links": []
+            },
+            "overrides": []
+          },
+          "fill": 1,
+          "fillGradient": 0,
+          "gridPos": {
+            "h": 8,
+            "w": 12,
+            "x": 0,
+            "y": 0
+          },
+          "hiddenSeries": false,
+          "id": 32,
+          "interval": "",
+          "legend": {
+            "avg": false,
+            "current": false,
+            "max": false,
+            "min": false,
+            "show": true,
+            "total": false,
+            "values": false
+          },
+          "lines": true,
+          "linewidth": 1,
+          "nullPointMode": "null",
+          "options": {
+            "alertThreshold": true
+          },
+          "percentage": false,
+          "pluginVersion": "8.2.6",
+          "pointradius": 2,
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "topk(10, \n  histogram_quantile(0.50, \n    sum(rate(\n      label_replace(\n        rest_client_request_duration_seconds_bucket{verb=\"GET\"},\n          \"url_host\",\"$1$2\",\"url\",\"https?://([^:^/]*)(:[0-9]+)?.*?\")[$period:30s]\n       )\n    ) by (le,url_host,job)\n  )\n)  ",
+              "format": "time_series",
+              "interval": "",
+              "legendFormat": "{{job}}-{{url_host}}",
```

Comment:
why is this failing here?

```
=== RUN   TestYamlCorrectness
    assets_test.go:34: Unexpected error reading manifests from ../../manifests/: failed to render "0000_90_kube-apiserver-operator_06_api_clients_dashboard.yaml": template: 0000_90_kube-apiserver-operator_06_api_clients_dashboard.yaml:95: function "job" not defined
```

Reply:
`"exemplar": true` seems incorrect here since we are not using exemplars with this histogram. Aside from that, I am not too sure, maybe the legendFormat requires values to be separated by whitespace? `{{job}} - {{url_host}}`